src/null/null_vm.cc

```diff
@@ -36,6 +36,13 @@ RegisterNullVmPluginFactory::RegisterNullVmPluginFactory(std::string_view name,
   (*null_vm_plugin_factories_)[std::string(name)] = factory;
 }
 
+Cloneable NullVm::cloneable() {
+  if (!plugin_) {
+    return Cloneable::NotCloneable;
+  }
+  return plugin_->cloneable();
+}
```

Comment:
`Cloneable` is a property of a runtime, so it shouldn't change based on whether or not the plugin was already loaded. It seems like a workaround for an issue elsewhere.

Reply:
agreed. Le me find more proper place to fix.

Reply:
https://github.com/mathetake/proxy-wasm-rust-nullvm/commit/ddfe09fb8914f93acc2439f00a6e486cc90856bc can be fixed in the glue code.